source/_posts/2018-05-31-automating-your-infrastructure-deployments.markdown

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: "Automating your infrastructure deployments"
+category: infrastructure
+comments: true
+author:
+  name: Federico Kauffman
+  email: federico@wyeworks.com
+  twitter_handle: fedekauffman
+  github_handle: fedekau
+  image: /images/team/federico-kauffman.jpg
+  description: Software Engineer at WyeWorks. Currently working with Javascript and Ruby. Learnaholic.
+---
+
+We all know that modern applications can get very complex very quickly. Because of that I've been trying to add more automation to my daily life in general, I know I am human and I know I will eventually fail because I was distracted, I was tired or something else.
+
+One of the firsts things I did in my automation journey was figuring out a way to automate the creation of cloud resources, to do that I became proficient at using some infrastructure as code tools like Terraform, AWS CloudFormation, and others.
+
+After that, since I have (or try to have) CI/CD pipelines setup in every project I am involved, my next logical step was to integrate that knowledge in infrastructure provisioning into the CI/CD pipelines. In this post I will be presenting benefits of doing that and will use an specific Terraform + CircleCI example.
+
+<!--more-->
+
+## Benefits of having automated infrastructure
+
+Let's start with the benefits of walking this path. You might be thinking: Automatically deploying infrastructure? This guy is crazy.
+
+If you are not used to this concept it might sound crazy, but in my opinion the benefits outweighs the risks. The main benefits include:
```

Comment:
the benefits outweigh~~s~~ the risks

Reply:
I didn't understand this comment...

Reply:
I meant it should be _outweight_ instead of _outweights_